ads/google/a4a/utils.js

```diff
@@ -172,6 +175,10 @@ export function googleAdUrl(
     if (isCanary(win)) {
       queryParams.push({name: 'isc', value: '1'});
     }
+    let eids = adElement.getAttribute('data-experiment-id');
```

Comment:
The point is to also capture existing ads on the page. So we could append them to the data attribute in the amp-ad-network-adsense-impl.js constructor, but potentially diverting an experiment in a constructor seems kind of nasty.

Another option would be to append them to the data attribute in the getUrl function in amp-ad-network-adsense-impl.js, but seems a bit strange for that function to have side-effects like that.